lib/sqlalchemy/engine/default.py

```
@@ -514,20 +514,20 @@ def initialize(self, connection):
             self.server_version_info = self._get_server_version_info(
                 connection
             )
-        except NotImplementedError:
+        except :
```

we can't suppress all errors here, those would need to be propagated